src/components/AmplitudeAnalytics/TraceEvent.tsx

```diff
@@ -57,8 +57,9 @@ function getEventHandlers(
 
   for (const eventHandlerName of keys) {
     eventHandlers[eventHandlerName] = (eventHandlerArgs: unknown) => {
-      // call child event handler with original arguments
-      child.props[eventHandlerName]?.apply(child, eventHandlerArgs)
+      // call child event handler with original arguments, must be in array
+      const args = Array.isArray(eventHandlerArgs) ? eventHandlerArgs : [eventHandlerArgs]
```

Comment:
need this line because ran into errors while implementing token selector events using traceEvent.

Reply:
Why is it sometimes passed as an array and sometimes not?

Reply:
if it's already an Array, we pass it directly, and if there is a single argument we need to send it in an array (or an error will occur)